airflow/jobs/scheduler_job.py

```diff
@@ -747,21 +747,16 @@ def _process_dags(self, dags: List[DAG], session=None):
 
         return tis_out
 
-    def _find_dags_to_process(self, dags: List[DAG], paused_dag_ids: Set[str]) -> List[DAG]:
+    def _find_dags_to_process(self, dags: List[DAG]) -> List[DAG]:
         """
         Find the DAGs that are not paused to process.
 
         :param dags: specified DAGs
-        :param paused_dag_ids: paused DAG IDs
         :return: DAGs to process
         """
         if len(self.dag_ids) > 0:
```

Comment:
```suggestion
        if self.dag_ids:
```
Should fix pylint error that will be found one day :)

Reply:
I prefer to limit additional changes if I make changes to the core. This makes the review more difficult.

Reply:
Done.